src/akushon/action/node/action_manager.cpp

```diff
@@ -160,6 +192,44 @@ void ActionManager::start(std::string action_name, const Pose & initial_pose)
   is_running = true;
 }
 
+void ActionManager::start(std::string action_name, std::string target_action_name, const Pose & initial_pose, float ball_x, float right_map_x_min_, float right_map_x_max_, float left_map_x_min_, float left_map_x_max_, bool right)
+{
+  std::vector<Action> target_actions;
+
+  while (true) {
+    auto & action = actions.at(action_name);
+    const auto & target_action = actions.at(target_action_name);
+
+    for (int pose_index = 0; pose_index < action.get_pose_count(); pose_index++) {
```

Comment:
Ditto:
```suggestion
    for (int pose_index = 0; pose_index < action.get_pose_count(); ++pose_index) {
```
Or better:
```suggestion
    for (int pose_index = action.get_pose_count() - 1; pose_index >= 0; --pose_index) {
```